helm/flowforge/templates/configmap.yaml

```diff
@@ -161,6 +161,8 @@ data:
       backend:
         sentry:
           dsn: {{ .Values.forge.telemetry.sentry.backend_dsn}}
+      prometheus:
+        enabled: {{ .Values.forge.telemetry.prometheus.enabled }}
```

Comment:
This won't get added unless `forge.telemtry.sentry` is also set (controlled by the `if` block above and the `end` under).

Also it looks like the `promethus` tag should be one level deeper (to match the depth of `sentry` under `backend`

I can't suggest a change as it would need to include changes to lines not added by this PR

Something like this should work:

```
      backend:
        {{ if and (hasKey .Values.forge.telemetry "sentry") (hasKey .Values.forge.telemetry.sentry "backend_dsn") -}}
        sentry:
          dsn: {{ .Values.forge.telemetry.sentry.backend_dsn}}
        {{- end }}
        prometheus:
          enabled: {{ .Values.forge.telemetry.prometheus.enabled }}
```

Reply:
I also think the denting of `prometheus` is one level too deep, it's not a subkey of `backend` but top level. Thanks for catching this. Pushed a change, let me know if this is correct @hardillb

Reply:
But the code in the forge app expects it to be under `backend`

https://github.com/FlowFuse/flowfuse/blob/fe0e996f9b9d8a14f7b1dc67d650e92734e5ae25/forge/forge.js#L81